packages/pnpmfile/src/requireHooks.ts

```diff
@@ -75,6 +78,7 @@ export default function requireHooks (
   } else {
     cookedHooks.filterLog = globalFilterLog ?? filterLog
   }
+  cookedHooks.importPackage = hooks.importPackage ?? globalHooks.importPackage
```

Comment:
This makes sense but if a user defines a custom `importPackage` hook it would result in the store hydration to be much slower. Not sure if there's something we can do or if it's their "fault" in that case.

Reply:
Why would someone create a custom importer?

Reply:
Yea that's a good question, probably not a valid use case I was considering.

Reply:
And if they do, they are on their own anyways.

Reply:
Did you mean install would be slower in the webcontainer if a project had a custom hook in a `.pnpmfile.cjs` committed?

Reply:
Yes, because it would then bypass the optimization that I am doing where I can link an entire project and its files with a single FS call.

Reply:
But it's fine I think. It will just be slower then if they have their own custom importer.